packages/vuetify/src/components/VMenu/VMenu.tsx

```diff
@@ -59,7 +59,7 @@ export const VMenu = genericComponent<OverlaySlots>()({
         ++openChildren.value
       },
       unregister () {
-        --openChildren.value
+        openChildren.value = Math.max(0, openChildren.value - 1)
```

Comment:
Having unregister sometimes called twice for the same item seems like it could cause other problems, I think a Set of uids needs to be used instead of just clamping a counter to 0.